libraries/rtos/rtx/TARGET_CORTEX_M/RTX_Conf_CM.c

```diff
@@ -168,6 +168,12 @@
 #elif defined(TARGET_STM32F401VC)
 #    define OS_CLOCK       84000000
 
+#elif defined(TARGET_NZ32ST1L)
+#    define OS_CLOCK       32000000
+
+#elif defined(TARGET_STM32L152RE)
```

Comment:
This pull request is for NZ32 target, and there are some TARGET_STM32L152RE, can you elaborate?